documentation/modules/configuring/proc-moving-node-pools.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// assembly-config.adoc
+
+[id='proc-moving-node-pools-{context}']
+= (Preview) Moving Kafka nodes between node pools
+
+[role="_abstract"]
+This procedure describes how to move nodes between source and target Kafka node pools without downtime.
+You create a new node on the target node pool and reassign partitions to move data from the old node on the source node pool.
+When the replicas on the new node are in-sync, you can delete the old node.
+
+The steps assume that you are using ZooKeeper for cluster management. 
+The operation is not possible for clusters using KRaft mode.
+
+NOTE: During this process, the ID of the node that holds the partition replicas changes. Consider any dependencies that reference the node ID.
+
+.Prerequisites
+
+* xref:deploying-cluster-operator-str[The Cluster Operator must be deployed.]
+
+.Procedure
+
+. Create a new node in the target node pool.
++
+For example, node pool `pool-a` has three replicas. We add a node by increasing the number of replicas:
```

Comment:
Can we have a short introduction here? About what we have (pool-a with 3 replicas and pool-b with 4 replicas) and what we want to achieve (moving to pool-a having 4 replicas and scale down pool-b to 3 and moving data).
As it is now, I am going to "discover" what's happening while doing. A short intro could set expectations better while reading imho.

Reply:
Good idea. I have added a summary to the intro.